shared/constants/network.ts

```diff
@@ -575,7 +575,7 @@ export const FEATURED_RPCS: RPCDefinition[] = [
     chainId: CHAIN_IDS.AURORA,
     nickname: AURORA_DISPLAY_NAME,
     rpcUrl: `https://aurora-mainnet.infura.io/v3/${infuraProjectId}`,
-    ticker: CURRENCY_SYMBOLS.AURORA,
```

Comment:
Default currency for Aurora is ETH, not the Aurora token: https://hackernoon.com/from-ethereum-to-aurora-with-infura